aexpect/remote.py

```diff
@@ -425,15 +444,17 @@ def wait_for_login(client, host, port, username, password, prompt,
         try:
             return remote_login(client, host, port, username, password, prompt,
                                 linesep, log_filename, log_function,
-                                internal_timeout, interface, verbose=verbose)
+                                internal_timeout, interface, verbose=verbose,
+                                preferred_authenticaton=preferred_authenticaton)
         except LoginError as error:
             logging.debug(error)
             verbose = True
         time.sleep(2)
     # Timeout expired; try one more time but don't catch exceptions
     return remote_login(client, host, port, username, password, prompt,
                         linesep, log_filename, log_function,
-                        internal_timeout, interface)
+                        internal_timeout, interface,
+                        preferred_authenticaton=preferred_authenticaton)
```

Comment:
We could then also call this the current `user_known_hosts_file` argument.

Reply:
Sure